README.md

```diff
@@ -70,9 +70,45 @@ should look something like this:
 -DVCPKG_TARGET_TRIPLET=x64-osx-static
 ```
 
+See the following images:
+
+### CLion ToolChain Settings
+
+![toolchain](images/toolchains.png?raw=true "Toolchains Settings")
+
+### CLion CMake Settings
+
+![CMake](images/cmake.png?raw=true "Cmake Settings")
+
 If you run `./run-genny install -b make` it should set up everything for you.
 You just need to set the "Generation Path" to your `build` directory.
 
+### Automatically Running Poplar before genny_core launch in CLion
+
+Create a file called poplar.sh with the following contents:
```

Comment:
I didn't even know this was a thing.
If you cut a TIG ticket I'd be happy to incorporate this into the `run-genny` CLI. I think you'd suggested a `run-genny curator` thing that would do this and that seems 💯

Reply:
[TIG-2934](https://jira.mongodb.org/browse/TIG-2934).